ct-app/core/components/hoprd_api.py

```
@@ -1,256 +1,178 @@
-        except MaxRetryError:
-            self.error(
-                f"MaxRetryError calling {obj.__name__}.{method} "
-                + f"with kwargs: {kwargs}, args: {args}"
-            )
+            headers = {"Content-Type": "application/json"}
+            async with aiohttp.ClientSession(headers=self.headers) as s:
+                async with getattr(s, method.value)(
+                    url=f"{self.host}{self.prefix}{endpoint}",
+                    json=data,
+                    headers=headers,
+                ) as res:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)